tests/migration/deregister_ltss.pm

```diff
@@ -0,0 +1,20 @@
+# SLE15 migration tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: LTSS is not supported to do migration, need to deregiter it before migration
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
```

Comment:
Not used functions from testapi, please remove this line.

Reply:
updated.